src/components/TextLink/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import styled from "lib/styled";
+import Text, { Props } from "components/Text";
```

Comment:
I think it makes sense here to create a new interface that extends the `Props` from the text component, since i can see this diverging soon to handle different cases we want to use an anchor. what do you think?

Reply:
Yeah that makes sense